eng/SmokeTestInputs.txt

```diff
@@ -67,13 +67,13 @@ https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b27
 #https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b274014245ae67/specification/cognitiveservices/data-plane/WebSearch/readme.md - https://github.com/Azure/autorest/issues/3498
 https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b274014245ae67/specification/cognitiveservices/resource-manager/readme.md
 #https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b274014245ae67/specification/commerce/resource-manager/readme.md - https://github.com/Azure/autorest.modelerfour/issues/257
-#https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b274014245ae67/specification/compute/resource-manager/readme.md - https://github.com/Azure/autorest.modelerfour/issues/257
+https://github.com/Azure/azure-rest-api-specs/blob/d9cf7c7ed3d674ebd482836e82b274014245ae67/specification/compute/resource-manager/readme.md
```

Comment:
One thing I wanted to mention, I thought that any smoke tests marked as `https://github.com/Azure/autorest.modelerfour/issues/257` would be able to be resolved now.

Reply:
Pretty sure they would, just don't want to block this PR for longer.